browser/ui/views/toolbar/brave_vpn_button.cc

```diff
@@ -187,6 +189,16 @@ void BraveVPNButton::UpdateColorsAndInsets() {
   SetImageLabelSpacing(kBraveAvatarImageLabelSpacing);
 }
 
+std::u16string BraveVPNButton::GetTooltipText(const gfx::Point& p) const {
+  auto text = l10n_util::GetStringUTF16(IDS_BRAVE_VPN);
+  if (!IsPurchased())
+    return text;
+
+  return text + u": " +
+         l10n_util::GetStringUTF16(IsConnected() ? IDS_BRAVE_VPN_CONNECTED
```

Comment:
Instead of combining two strings, how about using separated strings resources for conntected and disconnected tooltip.
Sometimes, combining translated two different strings make weird result.

Reply:
done